pkg/cli-runtime/testing/validatable_table.go

```diff
@@ -60,24 +72,31 @@ func (ts ValidatableTestSuite) Run(t *testing.T) {
 	}
 	if len(focused) != 0 {
 		for _, tc := range focused {
-			tc.Run(t)
+			tc.Run(t, ctx)
 		}
 		t.Errorf("test run focused on %d record(s), skipped %d record(s)", len(focused), len(ts)-len(focused))
 		return
 	}
 
 	for _, tc := range ts {
-		tc.Run(t)
+		tc.Run(t, ctx)
 	}
 }
 
-func (tc ValidatableTestCase) Run(t *testing.T) {
+func (tc ValidatableTestCase) Run(t *testing.T, ctx context.Context) {
 	t.Run(tc.Name, func(t *testing.T) {
 		if tc.Skip {
 			t.SkipNow()
 		}
 
-		errs := tc.Validatable.Validate(context.Background())
+		if tc.Prepare != nil {
+			var err error
+			if ctx, err = tc.Prepare(t, ctx); err != nil {
+				t.Errorf("error during prepare %v", err)
```

Comment:
stick to consistent error msgs. 
```sh
t.Errorf("unexpected error in _func_name_ : %v", err)
```

Reply:
https://github.com/vmware-tanzu/apps-cli-plugin/pull/402#discussion_r1025759155

Reply:
I sticked to the pattern mentioned in the previous comment